frontend/src/components/datasets/DatasetTable.tsx

```diff
@@ -45,6 +46,17 @@ const getTableColumns = () => {
             dataIndex: "lane",
             sorter: true,
         },
+        {
+            title: "Validated",
+            dataIndex: "validation_status",
+            render: (validation_status) => {
+                return (
+                    (validation_status === 1 && <Button style={{color: "#a0d911"}}><CheckOutlined onPointerEnterCapture={undefined} onPointerLeaveCapture={undefined}/>Passed</Button>)
```

Comment:
the onPointer... undefined callbacks look like defaults. Do they serve a purpose here ?

Reply:
they make my linter stop complaining :)

Reply:
in that case ... sorry mr linter ...